scienceplots/styles/color/discrete-rainbow/discrete-rainbow-1.mplstyle

```diff
@@ -0,0 +1,5 @@
+# Discrete rainbow color scheme with 1 colors
+# from Paul Tot's website: https://personal.sron.nl/~pault/
+
+# Set color cycle
+axes.prop_cycle : cycler('color', ['1965B0'])
```

Comment:
Can we add a new line at the end of the mplstyle files? Just to be style-consistent with the existing codebase.